changes to label group component design guidelines

pankajshivpuje opened the pull request:
Link to the updated Label group [design doc]

Here are the updated images:

(https://docs.google.com/document/d/1mSQlcssmhwOuiByR3Hx_5zfbZcTCzuH9DBs2oKqpdcg/edit#)
![editable-label-group-in-a-table_1](https://user-images.githubusercontent.com/40463673/208547979-9bbe117d-6e2e-4b1e-931b-5fb89afb0159.png)
![editable-label-group-in-a-table_2_1](https://user-images.githubusercontent.com/40463673/208547980-05cdb09e-4be8-43c7-bc70-dd57bdaa9b6f.png)
![editable-label-group-in-a-table_2](https://user-images.githubusercontent.com/40463673/208547982-4d353f32-5418-4999-8c25-8ca5e28f5dc3.png)
![editable-label-group-in-a-table_3_1](https://user-images.githubusercontent.com/40463673/208547984-d92712eb-1b67-42b1-968f-c22ee72d45e9.png)
![editable-label-group-in-a-table_3_2](https://user-images.githubusercontent.com/40463673/208547985-d82ef4b5-a79b-4ffb-859d-32e75a4578a4.png)
![editable-label-group-in-a-table_3](https://user-images.githubusercontent.com/40463673/208547986-a2d2c8ca-ec31-47e0-808e-09c4533b83f6.png)
![editable-label-group-in-a-table_4_1_add-label-model](https://user-images.githubusercontent.com/40463673/208547989-d46cbaea-d0ee-481d-93e6-9b715b8b9dc9.png)
![editable-label-group-in-a-table_4_2](https://user-images.githubusercontent.com/40463673/208547990-c23b4efe-7ea2-45c5-9c9a-18b153c4585d.png)
![editable-label-group-in-a-table_4](https://user-images.githubusercontent.com/40463673/208547993-b4b4ce66-1919-42e1-bfa5-045b6f3fa05a.png)

patternfly-build commented:
PF4 preview: https://patternfly-org-pr-3323-v4.surge.sh/v4

mcarrano commented on packages/v4/patternfly-docs/content/design-guidelines/components/label-group/label-group.md:
```diff
@@ -26,3 +26,40 @@ Label groups should be used when there are multiple labels assigned to a compone
+
+### Editable label group
+
+#### Adding a new label to a group, without a group label
```
I'm a bit confused about this variation. The title says "adding a new label," but the example seems to be about editing a label. I'd also spell out the word "one" here, as in "...there is a need to edit one or more labels."

pankajshivpuje replied:
@mcarrano I have updated the design guidelines doc, added an image and made textual changes to address your questions. I have also pushed the updated in the PR. Let me know if any additional changes are needed.

mcarrano reviewed (changes requested):
@pankajshivpuje this is looking good except for a couple of minor things. See my comments. I also think that it would help to use a callout to point to the action in some of the examples since it was hard to tell at a glance where one would click.

I know you will be out after this week, but if you make these changes today I will merge this as @mmenestr ison PTO until Jan 9.

pankajshivpuje commented:
@mcarrano I have updated the design guidelines doc, added an image and made textual changes to address your questions. I have also pushed the updated in the PR. Let me know if any additional changes are needed.

mcarrano reviewed (approved):
Looks great. Thanks for making those changes @pankajshivpuje . I will merge.
